services/sendgrid/src/handler.ts

```diff
@@ -0,0 +1,31 @@
+/* eslint @typescript-eslint/no-unsafe-assignment: 0 */
+/* eslint @typescript-eslint/no-unsafe-call: 0 */
+/* eslint @typescript-eslint/no-unsafe-member-access: 0 */
+import { sendEmail, SendEmailConfig } from '@src/send-email';
+import { SQSHandler } from 'aws-lambda';
+import aws from 'aws-sdk';
+
+export const main: SQSHandler = async (event, context, callback) => {
+  const { Records } = event;
+  const stepFunction = new aws.StepFunctions();
+  try {
+    await Promise.all(
+      Records.map(async (record) => {
+        const messageBody = JSON.parse(record.body);
+        const { taskToken } = messageBody;
+        const emailData = messageBody.payload as SendEmailConfig;
```

Comment:
it's probably this

Reply:
not sure what u mean by that

Reply:
For the lint-ignore.

Reply:
Top line of the file.

Reply:
oh I see what u mean. I already fixed that one. Not sure why it was still there though.